### PR TITLE
[zh] Migrated to use the v1beta3 API version of FlowSchema

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/40514/files

### CODE REVIEW

**Comment by tengqm** [comment]
should be fine but I can see more changes related to indentation in the English upstream.
Can we sync them together?

**Review by windsonsea — COMMENTED** [review]
Some indentations are changed too. Do we need to cover them?

image

**Comment by Arhell** [comment]
@tengqm @windsonsea done

File: content/zh-cn/examples/priority-and-fairness/health-for-strangers.yaml [hunk 1]
```diff
@@ -7,14 +7,14 @@ spec:
...
+        - nonResourceURLs:
+            - "/healthz"
+            - "/livez"
+            - "/readyz"
+          verbs:
+            - "*"
+      subjects:
+        - kind: Group
+          group:
+            name: "system:unauthenticated"
```
Comment (windsonsea) [inline content/zh-cn/examples/priority-and-fairness/health-for-strangers.yaml:20]:
```suggestion
    - nonResourceRules:
      - nonResourceURLs:
          - "/healthz"
          - "/livez"
          - "/readyz"
        verbs:
          - "*"
      subjects:
        - kind: Group
          group:
            name: "system:unauthenticated"
```

**Review by windsonsea — COMMENTED** [review]
Seems slightly different from en version:

```yaml
apiVersion: flowcontrol.apiserver.k8s.io/v1beta3
kind: FlowSchema
metadata:
  name: health-for-strangers
spec:
  matchingPrecedence: 1000
  priorityLevelConfiguration:
    name: exempt
  rules:
    - nonResourceRules:
      - nonResourceURLs:
          - "/healthz"
          - "/livez"
          - "/readyz"
        verbs:
          - "*"
      subjects:
        - kind: Group
          group:
            name: "system:unauthenticated"
```

Do we need to only change it in zh version? In my opinion, copy-paste is enough 😄

**Pull request merged**